src/components/Widgets/Markdown/serializers/remarkPaddedLinks.js

```diff
@@ -97,7 +97,9 @@ export default function remarkPaddedLinks() {
    * nesting. If `end` is truthy, get the last node, otherwise first.
    */
   function getEdgeTextChild(node, end) {
-    const findFn = end ? findLast : find;
+    let findFn;
```

Comment:
This really bothers me. I commented here to see if we can get some clarity: https://github.com/istanbuljs/babel-plugin-istanbul/issues/95.

Reply:
For now just add a comment explaining why it's written this way.